### PR TITLE
Kind upgrade to v0.8.1

### PR DESCRIPTION
**Pull request by nmiculinic**
**What this PR does / why we need it**:
Upgrades kind to v.0.8.1

```release-note
NONE
```

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -77,7 +77,7 @@ presubmits:
             - make e2e-test
           env:
             - name: KIND_NODE_IMAGE
-              value: "kindest/node:v1.17.0@sha256:9512edae126da271b66b990b6fff768fbb7cd786c7d39e86bdf55906352fdf62"
+              value: "kindest/node:v1.17.5@sha256:ab3f9e6ec5ad8840eeb1f76c89bb7948c77bbf76bcebe1a8b59790b8ae9a283a"
```
Comment (jiachengxu) [inline .prow.yaml:80]: Please also change the prow tasks names if you change the version here

File: .prow.yaml [hunk 2]
```diff
@@ -123,7 +123,7 @@ presubmits:
             - make e2e-test
           env:
             - name: KIND_NODE_IMAGE
-              value: "kindest/node:v1.16.4@sha256:b91a2c2317a000f3a783489dfb755064177dbc3a0b2f4147d50f04825d016f55"
+              value: "kindest/node:v1.16.9@sha256:7175872357bc85847ec4b1aba46ed1d12fa054c83ac7a8a11f5c268957fd5765"
```
Comment (jiachengxu) [inline .prow.yaml:126]: same for this

Comment (nmiculinic) [reply .prow.yaml:126]: done

File: .prow.yaml (hunk 1 above)
Comment (nmiculinic) [reply .prow.yaml:80]: done

File: .prow.yaml (hunk 2 above)
Comment (nmiculinic) [reply .prow.yaml:126]: done

File: .prow.yaml (hunk 1 above)
Comment (nmiculinic) [reply .prow.yaml:80]: done

**Pull request merged**